receiver/gitmetricsreceiver/README.md

```diff
@@ -0,0 +1,34 @@
+# Git Metrics Receiver
+
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [development]: metrics   |
+| Distributions | [] |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector#development
+<!-- end autogenerated section --> 
+
+The Git Metrics receiver generates metrics from Git about Git repositories and 
+organizations from various vendor endpoints. It is intended to be used
```

Comment:
Why would it only be "intended to be used when the collector is deployed as an agent"?

Reply:
This is a poor choice of wording on my part. Good catch. The sentence as a whole doesn't need to exist at all. Will remove.

Reply:
If we switch to `gitprovider` I'm thinking of switching this whole block to something very simple like:

> The Git Provider receiver scrapes data from git vendors.

Thoughts?